tests/pfcwd/test_pfcwd_function.py

```diff
@@ -529,7 +529,7 @@ def test_pfcwd_actions(self, request, setup_pfc_test, fanout_graph_facts, ptfhos
              logger.info("")
              logger.info("--- Testing various Pfcwd actions on {} ---".format(port))
              self.setup_test_params(port, setup_info['vlan'], init=not idx)
-             self.traffic_inst = SendVerifyTraffic(self.ptf, dut_facts['ansible_eth0']['macaddress'], self.pfc_wd)
+             self.traffic_inst = SendVerifyTraffic(self.ptf, dut_facts['ansible_Ethernet0']['macaddress'], self.pfc_wd)
```

Comment:
Not all platform HWSKU has Ethernet0 enabled. This is another loophole. We might want to read the config_db to get the mac address instead.

Reply:
Agree, config_db is better. I fixed this PR but apparently it is the same problem in a lot of tests

Reply:
Please, take a look at https://github.com/Azure/sonic-mgmt/pull/2552 which does similar change for all the pytest tests that are using ansible_Ethernet0